components/src/AppHeaderLogin/appheaderlogin.main.tsx

```diff
@@ -175,62 +175,66 @@ class AppHeaderLoginMain extends React.Component<AppHeaderLoginMainProps, AppHea
                                 <li className="dropdown-item">
                                     <Link to={appHeaderLoginLinks.profile} className="profile-link">
                                         <div>
-                      <span id="header_navbar_login_menu_profile_link">
-                        {intl.get('my-profile')}
-                      </span>
+                                          <span id="header_navbar_login_menu_profile_link">
+                                            {intl.get('my-profile')}
+                                          </span>
                                         </div>
                                     </Link>
                                 </li>
+                                {(Config.b2b.enable) ? (
+                                  <li className="dropdown-item">
+                                    <Link className="dashboard-link" to="/b2b">
+                                      <div>
+                                        <span className="dashboard-nav">
+                                          {intl.get('dashboard')}
```

Comment:
We need to confirm how this will be visible/hidden depending on the buyer's role. According to the requirement, this should only be displayed for Buyer Admins.